netty/src/main/java/io/grpc/netty/ProtocolNegotiator.java

```diff
@@ -44,6 +44,9 @@
 
   /**
    * The Netty handler to control the protocol negotiation.
+   *
+   * <p>TODO(carl-mastrangelo): Replace ChannelHandler with ChannelInboundHandler or
```

Comment:
It nags when building, and I did not know what the correct alternative is.  CIH adds a bunch of methods unrelated to how we are using CH here, so I left it as a todo.